src/libraries/System.Private.CoreLib/src/System/Number.BigInteger.cs

```diff
@@ -31,7 +31,9 @@ internal static partial class Number
             private const int MaxBits = BitsForLongestBinaryMantissa + BitsForLongestDigitSequence + BitsPerBlock;
 
             private const int BitsPerBlock = sizeof(int) * 8;
-            private const int MaxBlockCount = (MaxBits + (BitsPerBlock - 1)) / BitsPerBlock;
+
+            // We need one extra block to make our shift left algorithm significantly simpler
```

Comment:
Unrelated changes?

Reply:
Yeah, closing this one - will open new PR - thanks!